specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/InterfaceEndpointProfiles.json

```diff
@@ -0,0 +1,416 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-10-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/interfaceEndpointProfiles/{interfaceEndpointProfileName}": {
+      "get": {
+        "tags": [
+          "InterfaceEndpointProfiles"
+        ],
+        "description": "Gets a interface endpoint profile.",
+        "operationId": "InterfaceEndpointProfiles_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "interfaceEndpointProfileName",
+            "in": "path",
+            "description": "The name of the interface endpoint profile.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved a specified interface endpoint profile.",
+            "schema": {
+              "$ref": "#/definitions/InterfaceEndpointProfile"
+            }
+          },
+          "default": {
```

Comment:
add schema for the default response. it should be compliant with RPC error contract. See batch RP swagger for a sample

Reply:
discussed with @jaredmoo and @amolagar and found that this could be temporarily deprioritized